soh/soh/GameMenuBar.cpp

```diff
@@ -694,6 +694,9 @@ namespace GameMenuBar {
                     UIWidgets::PaddedEnhancementCheckbox("Remember Save Location", "gRememberSaveLocation", true, false);
                     UIWidgets::Tooltip("When loading a save, places Link at the last entrance he went through.\n"
                             "This doesn't work if the save was made in a grotto.");
+                    UIWidgets::EnhancementSliderInt("Fast Forward: %dx", "##FASTFORWARD", "gFrameMult", 1, 8, "", 1);
+                    UIWidgets::Tooltip("Speeds up gameplay. Note that some things, like text,\n"
+                                       "are unaffected by this: use the other enhancements to speed them up.");
```

Comment:
You don't need to manually specify newlines to break up long tooltips.  Also, I don't think you need to mention the "use other enhancements to speed them up" at all.

Reply:
Changes made